src/components/fields/localized-multiline-text-field/localized-multiline-text-field.visualroute.js

```diff
@@ -2,10 +2,13 @@ import React from 'react';
 import { LocalizedMultilineTextField } from 'ui-kit';
 import { Suite, Spec } from '../../../../test/percy';
 
+const lorem =
+  'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.';
+
 const value = {
-  en: 'hello\nworld',
```

Comment:
this stopped working in Percy after rebuilding yarn lock, and I coudln't figure out why.